lib/helper.ts

```diff
@@ -95,6 +95,20 @@ export function stripTrailingSlash(url: string): string {
   return url.replace(/\/$/, '');
 }
 
+/**
+ * Return a query parameter value from a URL
+ *
+ * @param {string} urlStr - the url string.
+ * @param {string} param - the name of the query parameter
+ *                     whose value should be returned
+ * @returns {string} the value of the `param` query parameter
+ * @throws if urlStr is an invalid URL
+ */
+export function getQueryParam(urlStr: string, param: string): string {
+  const url = new URL(urlStr, 'https://foo.bar');
```

Comment:
Do you mind adding a comment explaining that `'https://foo.bar'` is there to support relative URLs? Maybe this is more common than I realize but I've never seen that pattern and I was a little confused. Upon looking it up it made perfect sense but a comment may be helpful for future people

Reply:
Good suggestion!  I will add a comment.